docs/INSTALLATION.md

```diff
@@ -1,6 +1,6 @@
 ## Installation
 
-Use the [installation guide at Jumpstart Lab](http://tutorials.jumpstartlab.com/topics/environment/environment.html) for instructions on installing Ruby.
+Choose your installation method at the [Ruby installation page](https://www.ruby-lang.org/en/documentation/installation/) for instructions on installing Ruby.
```

Comment:
Would it make sense to recommend a method here? If I was installing Ruby for the first time, I'd like for someone to recommend me `rbenv` for example

Reply:
You mean on the official site?  I think it does recommend methods there for different situations, operating systems, etc.

You wouldn't recommend `rbenv` for Windows users, I think.  Maybe you would.  I wouldn't recommend `chruby` for Linux users.

The users know their systems better than we do, I think the official page should address the ways that it can be installed on various systems.  They are vested in that information.

Reply:
> The users know their systems better than we do
Good point, agreed.